lms/static/scripts/frontend_apps/components/ErrorDialogApp.js

```diff
@@ -0,0 +1,65 @@
+import { Fragment, createElement } from 'preact';
+import { useContext } from 'preact/hooks';
+
+import { Config } from '../config';
+
+import ErrorDisplay from './ErrorDisplay';
+import Dialog from './Dialog';
+
+/**
+ * Error dialog displayed when authorization with a third-party API via OAuth
+ * fails.
+ *
+ */
+export default function ErrorDialogApp() {
+  const {
+    ErrorDialog: { errorCode, errorDetails = '' },
+  } = useContext(Config);
+
+  const error = { code: errorCode, details: errorDetails };
+
+  const title = (() => {
+    return (
+      {
+        reused_tool_guid:
+          'The Hypothesis credentials your install is using may have already been used in a different LMS instance',
+      }[errorCode] || 'An error occurred'
+    );
+  })();
+
+  const message = (() => {
+    return (
+      {
+        reused_tool_guid: 'Reused tool_consumer_instance_guid',
+      }[errorCode] || 'Unknown error occurred'
+    );
+  })();
+
+  return (
+    <Dialog title={title}>
+      {error.code === 'reused_tool_guid' && (
+        <Fragment>
+          This may be due to one of the following:
+          <ul>
+            <li>
+              The same Hypothesis Consumer Key and Shared Secret has been used
+              previously in another LMS instance
+            </li>
+            <li>
+              The{' '}
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://www.imsglobal.org/specs/ltiv1p1p1/implementation-guide"
+              >
+                tool_consumer_instance_guid
+              </a>{' '}
+              of your LMS has changed
+            </li>
+          </ul>
+        </Fragment>
+      )}
+      <ErrorDisplay message={message} error={error} />
+    </Dialog>
+  );
+}
```

Comment:
I'll update the copy based on the lastest from https://hypothes-is.slack.com/archives/C2BLQDKHA/p1629287584304700